### PR TITLE
Add kaggle user agent to gcp integration clients.

### PR DESCRIPTION
**Pull request by vimota**

### CODE REVIEW

**Comment by vimota** [comment]
This works for all integrations except the automl TablesClient, they have a bug in their client initialization where if you provide a client_info= parameter it throws `TypeError: type object got multiple values for keyword argument 'client_info'`.

Cause of it is this line, where they get a client_info from kwargs, but then pass it as a parameter again, so it's passed in twice (as a named parameter and in the original kwargs):
https://github.com/googleapis/google-cloud-python/blob/9e5ca9a6e468d246e3aab5392ef0eeb4e802ec3d/automl/google/cloud/automl_v1beta1/tables/tables_client.py#L109

This should be easy to fix, I'll sent them an PR/Issue for it.

**Comment by rosbo** [comment]
Note, the `master` build is failing and is likely the cause of this failure. I will try fixing it today. Once `master` is green, you will have to rebase from master to get all the latest changes and make sure the build is green. Thanks

**Comment by rosbo** [comment]
I have fixed the `master` build. It should be good now.

File: patches/kaggle_gcp.py [hunk 1]
```diff
@@ -183,11 +186,28 @@ def patched_init(self, *args, **kwargs):
         if specified_credentials is None:
             Log.info("No credentials specified, using KaggleKernelCredentials.")
             kwargs['credentials'] = kaggle_kernel_credentials
+        
+        # TODO(vimota): Remove the exclusion of TablesClient once
+        # the client has fixed the error:
+        # `multiple values for keyword argument 'client_info'``
```
Comment (mcollins42) [inline patches/kaggle_gcp.py]: Is that an actual error that you see if this is set? I was hoping that Python would just ignore the kwargs value if a named value was already passed.

Comment (vimota) [reply patches/kaggle_gcp.py]: Yeah unfortunately it's a type error :( https://stackoverflow.com/questions/21764770/typeerror-got-multiple-values-for-argument

**Comment by vimota** [comment]
> I have fixed the `master` build. It should be good now.

Thanks, rebased and rerunning!

**Pull request merged**